app/controllers/admin/userManagement.py

```diff
@@ -65,7 +65,7 @@ def deleteProgramFile():
 
 @admin_bp.route('/admin/updateProgramInfo/<programID>', methods=['POST'])
 def updateProgramInfo(programID):
-    if g.current_user.isCeltsAdmin:
+    if g.current_user.isCeltsAdmin or g.current_user.isProgramManagerFor(Program.get_by_id(programID)):
```

Comment:
You can just pass `programID` to the method, rather than doing a database query to get a `Program` object and passing that.